### DISCUSSION TITLE
页面显示调整

### DISCUSSION THREAD

**Discussion by JCC-qiu** [CefView/QCefView#375] [Q&A]
[//]: # (以下所有的以"[//]: #"开始的文字都是注释，不会被markdown渲染，所以无需删除)
[//]: # (All lines start "with [//]: #" below are comment lines and will not be rendered by markdown, so there's no need to delete them)

** Feature Description 【特性详细描述】**
点击网页链接时，如何设置弹出的窗口全屏显示？

**Comment by L-Super** [comment]
点击网页链接后，函数会进入[onNewBrowser](https://cefview.github.io/QCefView/zh/docs/reference/QCefView#class_q_cef_view_1a0aa0b257b5383428569d46c39d4003a2) 或 [onNewPopup](https://cefview.github.io/QCefView/zh/docs/reference/QCefView#class_q_cef_view_1aff9055d5f9f773c2582c20b4bdf0cd33)，你可重写这两个函数，实现对应的功能，如打开新标签页或禁止打开新标签页，以及全屏也是自己能控制的
